clang/lib/StaticAnalyzer/Checkers/ArrayBoundCheckerV2.cpp

```diff
@@ -22,23 +22,25 @@
 #include "clang/StaticAnalyzer/Core/PathSensitive/DynamicExtent.h"
 #include "clang/StaticAnalyzer/Core/PathSensitive/ExprEngine.h"
 #include "llvm/ADT/SmallString.h"
+#include "llvm/Support/FormatVariadic.h"
 #include "llvm/Support/raw_ostream.h"
 #include <optional>
 
 using namespace clang;
 using namespace ento;
 using namespace taint;
+using llvm::formatv;
 
 namespace {
+enum OOB_Kind { OOB_Precedes, OOB_Exceeds, OOB_Taint };
```

Comment:
FYI you can use any identifiers here because these are only exposed to this single TU.
The ugly prefixes are only applied to enums exposed in headers etc. where there is an unbounded chance of having collisions.
No action is expected.

Reply:
The enumerator naming style was "inherited" from old versions of the checker. Perhaps I'll rename them later when I need to touch them for some reason.